admin/makedist.sh

```diff
@@ -45,7 +45,7 @@ for i in $*; do
 				DOMD5SUM="yes"
 			;;
 		--*)
-			echo "ERROR: Unrecognized option $i"
+			echo "ERROR! Unrecognized option $i"
```

Comment:
And this is just an internal script, I don't understand why would you change things just for the sake of it... again.